### DISCUSSION TITLE
Google Lens 研發辨識手寫處方箋功能

### DISCUSSION THREAD

**Discussion by liangyinglly** [1101-GS3073-C/Python-Community-GS3073-C#660] [General]
許多醫生在開處方箋時常會寫出潦草的手寫字，現在 Google 希望藉 AI 來破解這些難以閱讀的手寫處方，在印度舉辦的會談上宣布將是 Google Lens 的新應用之一。

我覺得由於醫生常有自己的一套速記法，且在處方箋的辨識上更是不能出錯，所以Google Lens應該還有很大的進步空間，而且我覺得這個功能其實蠻廢的，哈哈，病人可以直接向醫生詢問就好了啊。

我自己的想法：我自己覺得「藥品辨識」這種功能還蠻不錯的（不太確定現在有沒有這樣的程式或功能），因為我爺爺常常會拍藥罐或藥盒問我這是什麼藥，他的功能是什麼，所以我覺得如果有一種軟體是能透果掃描藥盒或藥罐上的英文或拉丁字母，就能立即顯示這款藥的功效與服用方式，對老人家來說，應該是很不錯的發明！

文章連結：https://www.inside.com.tw/article/30173-google-lens-doctors-handwritting

**Comment by bww0401** [comment]
雖說這個發明似乎幫助不大，但其延伸相關應用有很多，希望未來有更多醫療上的應用被開發出來!

**Comment by 0212Jerry** [comment]
這是一個有趣的發明，不過藥品的辨識需要極高的精確度，倘若一個疏失可能就是一條人命。因此，這項產品可能會需要大量的範本進行模擬與訓練。

**Comment by alvin1226** [comment]
現在大部分年輕醫生很少會寫處方簽了，大部分都直接打在電腦上，可能這項技術以後也用不到了QQ
